QUEST_LV_0100.tsv

```diff
@@ -6946,32 +6946,32 @@ QUEST_LV_0100_20150714_006945	A monster in disguise as a Goddess Statue. I pity
 QUEST_LV_0100_20150714_006946	We didn't just stay put when we were besieged by the monsters. {nl}We sent out Scouts to get away from here but no one came back. {nl}
 QUEST_LV_0100_20150714_006947	If you are okay, I want you to search for them.{nl}
 QUEST_LV_0100_20150714_006948	$It's going to be hard.{nl}The Thorny Forest keeps growing.
-QUEST_LV_0100_20150714_006949	$A dark aura similar to the voice a while ago is covering the thorny forests. {nl}Especially Kvailas Forest??Please be careful??
+QUEST_LV_0100_20150714_006949	$A dark aura similar to the voice a while ago is covering the thorny forests. {nl}Especially Kvailas Forest.. Please be careful.
 QUEST_LV_0100_20150714_006950	Good. {nl}First, defeat the Big Blue Grivas up the hills.
 QUEST_LV_0100_20150714_006951	When you throw this bomb to the long branch trees, they will burn.{nl}When you feel they are burnt enough, destroy those tress to get wood coals.{nl}
 QUEST_LV_0100_20150714_006952	$Be careful. This place is full of evil energy from Bramble.{nl}You better go back to be safe.
 QUEST_LV_0100_20150714_006953	$Ah, it was you, Revelator! Could you help me little more?{nl}I am trying to burn down the thorny bushes that are full of evil energy.
 QUEST_LV_0100_20150714_006954	There's a demon that I want you to defeat.{nl}The monster is called Likaous at Thorny Pole Garden. It brings vicious atmosphere with him.
-QUEST_LV_0100_20150714_006955	$I heard a very ominous voice looking for you.{nl}Please be careful??
+QUEST_LV_0100_20150714_006955	$I heard a very ominous voice looking for you.{nl}Please be careful..
 QUEST_LV_0100_20150714_006956	$I never thought I would meet someone other than a believer in a place full of thorny bushes and the monsters.{nl}It would be hard without the protection of the Goddess. Don't you think so, Revelator?{nl}
 QUEST_LV_0100_20150714_006957	$So you are the Revelator. The believers are having a hard time due to the thorny bushes.{nl}Can you help them??
 QUEST_LV_0100_20150714_006958	$The evil force was first felt in Kvailas Forest. {nl}The force was very strong so it will remain this way for a while even if you defeat Bramble.
-QUEST_LV_0100_20150714_006959	$The evil force of Kvailas Forest is becoming stronger but our followers are becoming weak. {nl}I am worried that we may lose to the forces of evil. If only the Revelator can be with us??
+QUEST_LV_0100_20150714_006959	$The evil force of Kvailas Forest is becoming stronger but our followers are becoming weak. {nl}I am worried that we may lose to the forces of evil. If only the Revelator can be with us..
```

Comment:
I have changed this line in my PR as well, however, mine is:

"The evil aura of Kvailas Forest is becoming stronger and our followers are becoming weak. {nl}I am worried that we may lose to the forces of evil. If only the Revelator can be with us..."